master/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -5,52 +5,46 @@ canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/i
 
 ## About Canal
 
-Canal allow users to easily deploy Calico and flannel networking
-together as a unified networking solution—combining Calico’s
+Canal allows users to easily deploy {{site.prodname}} and flannel networking
+together as a unified networking solution—combining {{site.prodname}}’s
 industry-leading network policy enforcement with the
 [flannel](https://github.com/coreos/flannel#flannel)
 overlay and non-overlay network connectivity options.
 
-Note that the Canal currently uses the Calico and flannel projects as is with
+Note that Canal currently uses the {{site.prodname}} and flannel projects as is with
 no code modifications to either. Canal is simply a deployment pattern
 for installing and configuring the projects to work together seamlessly as a
 single network solution from the point of view of the user and orchestration
 system.
 
 
-## Installation
+## Before you begin 
 
-When deploying your Kubernetes cluster, please make sure it meets the
-[requirements](#requirements--limitations) at the bottom of this page.
-An easy way to create a cluster which meets these requirements is by following
-[the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+- Ensure that your cluster meets the [requirements](../../../reqs/system).
```

Comment:
"meets the requirements" is a bit awkward... perhaps "Ensure your cluster meets the requirements to install canal"?

Reply:
or "meets these requirements" rings a lot better, actually.